modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -879,40 +1188,49 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::computeStressFinalize(
 {
   _cell_dislocation_increment = 0.0;
   _wall_dislocation_increment = 0.0;
-
-  precomputeROM(_cell_output_index);
-  for (unsigned int t = 0; t < _num_tiles; ++t)
-    if (_weights[t])
-      _cell_dislocation_increment += _weights[t] * computeROM(t, _cell_output_index);
-
-  precomputeROM(_wall_output_index);
-  for (unsigned int t = 0; t < _num_tiles; ++t)
-    if (_weights[t])
-      _wall_dislocation_increment += _weights[t] * computeROM(t, _wall_output_index);
+  if (_input_values[_stress_input_index])
```

Comment:
This fixed up the `nan`'s I was seeing @dschwen , not sure if it was the same for you or similar to your fix?